starter-code/iteration3/iteration3/src/app/components/nba-champions/nba-champions.component.html

```diff
@@ -0,0 +1,13 @@
+<section class="teams" >
+ <div *ngFor = "let champions of nbaChampions" class="col"> 
+    <h1>{{champions.year}}</h1>
+    <h3>{{champions.name}}</h3>
+    <img src="{{champions.championUrl}}">
+    <h3>Best Players</h3>
+    <ul *ngFor="let bestPlayers of champions.bestPlayers">
+      <li>{{bestPlayers}}</li>
+    </ul>
```

Comment:
```
<ul>
   <li *ngFor="let bestPlayers of champions.bestPlayers"> 
       {{bestPlayers}}
   </li>
</ul>
```

Reply:
Otherwise, if you inspect youll see that it is creating a lot of <ul> with one <li> inside